qwandaq/src/main/java/life/genny/qwandaq/entity/search/SearchEntity.java

```diff
@@ -599,6 +599,16 @@ public SearchEntity convertToSendable() {
 					ea.setIndex(i);
 				});
 
+		// add associated columns attributes
+		IntStream.range(0, associatedColumns.size())
```

Comment:
"Stream" might be an overkill if we are sequentially looping without using the actual stream benefits. This is not stopping us from moving forward though.